src/librustdoc/html/markdown.rs

```diff
@@ -220,11 +225,28 @@ impl<'a, I: Iterator<Item = Event<'a>>> Iterator for CodeBlocks<'a, I> {
                     url, test_escaped, channel
                 ))
             });
+            let title = if ignore {
+                let mut tmp = HashMap::new();
+                tmp.insert("title".to_owned(),
+                           "Be careful when using this code, it's not being tested!".to_owned());
+                Some(tmp)
+            } else if compile_fail {
+                let mut tmp = HashMap::new();
+                tmp.insert("title".to_owned(),
+                           "This code doesn't compile so be extra careful!".to_owned());
```

Comment:
:bike::derelict_house:: I'm wondering if there's a nicer way to phrase this than "so be extra careful" (since it doesn't compile anyway), but: I don't have a better suggestion than just leaving that part off so I think it's fine as-is, too.

Reply:
Something like "Warning: This code will not compile!", maybe?